index.html

```diff
@@ -75,4 +79,4 @@
 
 </body>
 
```

Comment:
you don't have to do this for this commit, but you will need the following frontend js code:

1) upon selecting a tool, your cursor should become that tool
2) upon selecting a tool, the corresponding tool button should still be "indented" (darker grey)